Update Interface (#4334)
ppy/osu-wiki: wiki/Interface/en.md

This is your online leaderboard. You can go here by scrolling down from the results screen. Your Local Scoreboard will show your name and the score as usual.

1. Your player bar. It shows your [PP](/wiki/Performance_Points), Global Rank, Total Score, Overall [Accuracy](/wiki/Accuracy), and level bar.

> 2. Save replay to Replays folder: That replay can be used offline.

Reviewer: ```suggestion
2. `Save replay to Replays folder`: You can watch the replay later either by opening it from a local leaderboard, or by going to `Replays` directory and double clicking it.
```

> 3. Online Favourite: Make the beatmap as your favourite. This will be visible in your osu! page, section "Beatmaps".

Reviewer: ```suggestion
3. `Add as online favourite`: Include the beatmap into your list of favourites, which is located on your osu! page under "Beatmaps" section.
```

> 4. Local Leaderboard: All your results are stored on the computer. To go to them, enter the [song selection screen](#song-selection-screen), you should see the [rankings](#rankings). Click on the drop-down menu and select "Local Rankings".

Reviewer: ```suggestion
4. Local Leaderboard: All your results are stored on your computer. To see them, navigate to the [song selection screen](#song-selection-screen), then select `Local Rankings` from the drop-down menu on the left.
```

> 5. Beatmap Ranking section. Will not appear if offline, or if the beatmap is not uploaded.
>    1. Overall: Your Ranking on the map. The result includes replays that featured mods, even if you didn't use any yourself.
>    2. Accuracy: How [precisely](/wiki/Accuracy) did you play the beatmap. Will only be counted when your old score is surpassed.
>    3. Max Combo: Your biggest combo on the map you played. If it's an unranked song, it will not increase at all.   
>    4. Ranked Score: Your [best result](/wiki/Score#ranked-score) on map. If it's an unranked song, it will not increase at all.
>    5. Total Score: Not taken into account, since it does not take a fate.
>    6. Performance: The received number of PP on map you played. If it's an unranked (or an approved) song, it will not increase at all.

Reviewer: ```suggestion
5. `Beatmap Ranking` section. Available only for maps with online leaderboards ([qualified](wiki/Beatmaps/en.md#qualified), [ranked](wiki/Beatmaps/en.md#ranked), or [loved](wiki/Beatmaps/en.md#loved)).
   1. `Overall`: Your position on the map's leaderboard, where you compete against players that used [mods](/wiki/Game_modifier/en.md), even if you didn't use any yourself.
   2. `Accuracy`: How [precisely](/wiki/Accuracy) did you play the beatmap. Will only be counted when your old score is surpassed.
   3. `Max Combo`: Your longest combo on the map you played.
   4. `Ranked Score`: Your [best result](/wiki/Score#ranked-score) on the map.
   5. `Total Score`: Not taken into account, since it does not affect your position in online rankings.
   6. `Performance`: The amount of [unweighted PP](/wiki/Performance_points/en.md#why-didnt-i-gain-the-full-amount-of-pp-from-a-map-i-played) you would receive for the play.
```

> 6. Overall Ranking section. It will also not be displayed offline, or if the beatmap is not uploaded.
>    1. Overall: Your global ranking in the world.
>    2. Accuracy: Your average accuracy obtained from the calculations of the maps you played.
>    3. Max Combo: Biggest combo on your profile (Calculated based on the largest combo in your maps).
>    4. Ranked Score: The number of points earned from all ranked maps that you have ever played on your profile.
>    5. Total Score: Same as #6.4 but it takes into account all types of ranking (Loved, Ranked, Approved, Unranked), underplayed and failed maps. Not submitted beatmaps are not counted.
>    6. Perfomance: The total amount of PP in your profile.

Reviewer: ```suggestion
6. `Overall Ranking` section. It's also available only for maps with online leaderboards.
   1. `Overall: Your global ranking in the world.
   2. `Accuracy`: Your [average accuracy](/wiki/Accuracy/en.md#accuracy) over all maps you have played.
   3. `Max Combo`: The longest combo over all maps you have played.
   4. [`Ranked Score`](/wiki/Score/en.md#ranked-score): The number of points earned from all ranked maps that you have ever played, with every map being counted exactly once.
   5. [`Total Score`](/wiki/Score/en.md#total-score): Same as ranked score, but it takes into account all types of ranking (Loved, Qualified, Ranked, Approved), underplayed and failed maps.
   6. `Perfomance`: The total amount of PP in your profile.
```

This is your online leaderboard. You can go here by scrolling down from the results screen. Your Local Scoreboard will show your name and the score as usual.

1. Your player bar. It shows your [PP](/wiki/Performance_Points), Global Rank, Total Score, Overall [Accuracy](/wiki/Accuracy), and level bar.

> 2. Export Replay as `.osr`: Save the replay that can be used online.

Reviewer: The caption is different ("Save replay to Replays folder"), and I also believe that a replay is saved for **offline** use

> 3. Online Favourite: Make the Beatmap as your Online Favourite. It will show on

Reviewer: - *Beatmap* is lowercase;
- *... It will show on* — where? not to mention "online favourite" should be in sentence case

> 4. Local Rankings: All your results are saved in local rankings

Reviewer: that's just restating what is there on the screenshot without any explanation, doesn't look useful

> 5. Self-explanatory (Beatmap Ranking). Will not appear if offline, or if the beatmap is not uploaded.

Reviewer: again, if it's self-explanatory, why restate it?

   1. Map Rank: Your Ranking on the map. The ranking includes replays that featured mods, even if you didn't use any yourself.
   2. Overall: Your profile standing on the gamemode you just played, not just the beatmap.
   3. Accuracy: Will only be counted when your old score is surpassed. Accuracy is the average of all passed beatmaps.

>    4. Ranked Score: Your Ranked Score on map. If it's an unranked song, it will not increase at all.

Reviewer: perhaps a link to the scoring concepts would be useful here. besides, again, this is just a repetitive statement: "Ranked Score: Your Ranked Score on map". even below, for the overall ranking, you give at least a brief explanation of what ranked score is

   5. Max Combo: Your maximum combo on the map you played. If it's an unranked song, it will not increase at all.
   6. Total Score: Grand total score. This will always increase even if you fail or don't surpass a high score in a beatmap. This counts towards your level.

>    7. Performance: The received number of PP (Performance Points) on map you played. If it's an unranked (or an approved) song, it will not increase at all.

Reviewer: as you explained what PP is at the beginning of the list, you can omit the full name of it
```suggestion
   7. Performance: The received number of PP on map you played. If it's an unranked (or an approved) song, it will not increase at all.
```

6. Self-explananatory (Overall Ranking):
   1. Overall: Your global ranking in the world.
   2. Accuracy: Your average accuracy obtained from the calculations of the maps you played.
   3. Max Combo: Max Combo on your profile (Calculated based on the largest combo in your maps).
   4. Ranked Score: The number of points earned from all ranked maps that you have ever played on your profile.
   5. Same as #6.4 but it takes into account all types of ranking (Loved, Ranked, Approved, Unranked), underplayed and failed maps. Not submitted beatmaps is not taken into account.

>    6. The total amount of pp in your profile. In depending of the received result, it can increase or decrease.

Reviewer: ```suggestion
   6. The total amount of PP in your profile. Depending of the received result, it can increase or decrease.
```
